sass/form/tools.sass

```diff
@@ -201,6 +201,16 @@ $label-colors: $form-colors !default
     .icon.is-right
       right: 0
   &.is-loading
+    &.has-icons-right
+      & > .icon.is-right
+        & > i
```

Comment:
Is this specific required here? It’s possible that with some icon framework the `<i>` is not used.

Reply:
in fact it's not necessary, I have already done the removal